patches/chromium/enable_partition_alloc_ref_count_size.patch

```diff
@@ -0,0 +1,34 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: VerteDinde <vertedinde@electronjs.org>
+Date: Wed, 13 Dec 2023 12:53:11 -0800
+Subject: fix: enable partition alloc ref count size for MacOS 13+
+
+Modifies an existing check to check for all versions of MacOS 13
+or higher, including MacOS 14 and potential beta versions.
+
+This patch is currently being added upstream, and can be removed
+when we integrate the upstream changes into Electron.
```

Comment:
Should include a link, if available

Reply:
Agreed - I went looking and wasn't able to find an existing CL, but I know a version is incoming. I'll keep an eye out and link for it when I see it 👍